src/main/java/org/springframework/data/redis/cache/DefaultRedisCacheWriter.java

```diff
@@ -59,6 +60,9 @@
  */
 class DefaultRedisCacheWriter implements RedisCacheWriter {
 
+	private static final boolean REACTIVE_REDIS_CONNECTION_FACTORY_PRESENT = ClassUtils
+			.isPresent("org.springframework.data.redis.connection.ReactiveRedisConnectionFactory", null);
```

Comment:
This is very ugly line-break. It read nicer as...

`ClassUtils.isPresent(..)`

on a single line.